indico/modules/events/registration/client/js/form_submission/RegistrationFormSubmission.jsx

```diff
@@ -175,6 +175,7 @@ export default function RegistrationFormSubmission() {
     >
       {fprops => (
         <form onSubmit={fprops.handleSubmit}>
+          {renderPluginComponents('regformBeforeSections')}
```

Comment:
Oh great, looks like we have a mix of  camelCase and snake-case for this...

Reply:
similar to
` {renderPluginComponents('regformAfterSections')}`
- https://github.com/indico/indico/pull/6218/files#diff-531ba68586d2b7bbb75806adb77fcc16170e1a61fbeaa65b37924b0e947b7941L181

Reply:
yeah, i think i used camelcase in the very first ones i added, and then some others were added as snake-case and i didn't realize we had camelCase here...

Reply:
Thanks for the prompt merge :)